smbclient/shutil.py

```diff
@@ -62,6 +62,16 @@
 )
 
 
+def is_remote_path(path) -> bool:
```

Comment:
For now you can do. I probably want to have 1 more release that supports 2.7

```suggestion
def is_remote_path(path):  # type: (str) -> bool
```